data/json/recipes/basecamps/recipe_primitive_field.json

```diff
@@ -1035,10 +1035,10 @@
     "blueprint_requires": [ { "id": "not_an_upgrade" } ],
     "time": "180 m",
     "blueprint_needs": {
-      "time": "1 d 6 h",
+      "time": "1 d 16 h",
       "skills": [ [ "survival", 3 ] ],
       "inline": {
-        "tools": [ [ [ "elec_jackhammer", 42000 ], [ "jackhammer", 840 ], [ "pickaxe", -1 ] ] ],
+        "tools": [ [ [ "elec_jackhammer", 56000 ], [ "jackhammer", 1120 ], [ "pickaxe", -1 ] ] ],
```

Comment:
Reason for this change?

Reply:
Something I changed required it.  I just encoded the newly autocalculated requirements.  I was a bit surprised and perhaps should have looked into it further.  If anything, I'd expect the requirements to have gone down from these changes so I'm not sure what happened.